examples/mallocMC_example03.cpp

```diff
@@ -65,49 +60,60 @@ struct ScatterHashParams
     static constexpr auto hashingDistWPRel = 1;
 };
 
-// configure the AlignmentPolicy "Shrink"
 struct AlignmentConfig
 {
     static constexpr auto dataAlignment = 16;
 };
 
-// Define a new mMCator and call it ScatterAllocator
-// which resembles the behaviour of ScatterAlloc
 using ScatterAllocator = mallocMC::Allocator<
+    Acc,
```

Comment:
I need the alpaka Acc for:
* the type of the buffers inside mallocMC::Accelerator. I could typ erase them maybe.
* the call to `alpaka::kernel::createTaskKernel` inside `initHeap` in the creation policy. If I can get the accelerator type from the alpaka device or queue, I could solve this otherwise. Is this possible?

Reply:
Ok keep it as it is for now. I will have a look into it when I test it with PIConGPU.